integrations/opencost/integration.py

```diff
@@ -134,6 +134,13 @@ class CloudCostSelector(Selector):
                 description="Step size of the accumulation.",
             )
         )
+        filter: str | None = Field(
+            description=(
+                "Filter results by any category which that can be aggregated by,"
+                " can support multiple filterable items in the same category in"
+                " a comma-separated list."
+            ),
```

Comment:
I think we should support list object and parse it to string

Reply:
We want to support this format:
```
filter:
  service: AWSSecretsManager,AWSS3,ECS
  provider: AWS
```